### ISSUE TITLE
frontend tests passing but reporting SyntaxError|TypeError about testInvalidPasswordAndValidationPassword

### ISSUE THREAD

**Issue by jrgm** [mozilla/persona#2972]
I was running tests locally on the dev branch and noticed this output. However the tests report as passed overall ($? == 0).

/me punts to shane

I also see the same chirps here: https://travis-ci.org/mozilla/browserid/jobs/4367739

Locally:

```
$ WHAT_TESTS=front npm test

> browserid@1.0.0-b1 test /Users/jmorrison/github/jrgm/browserid
> ./scripts/test

>>> Now Running: Front end unit tests under PhantomJS
· · · · router (6030): info: forwarding request: /common/js/error-messages.js -> http://127.0.0.1:10010/common/js/error-messages.js
SyntaxError: Parse error

TypeError: 'undefined' is not a function (evaluating 'testHelpers.testInvalidPasswordAndValidationPassword')

  http://127.0.0.1:10002/test/cases/pages/js/forgot.js:121
  http://127.0.0.1:10002/test/cases/pages/js/forgot.js:152
TypeError: 'undefined' is not a function (evaluating 'testHelpers.testInvalidPasswordAndValidationPassword')

  http://127.0.0.1:10002/test/cases/pages/js/signin.js:286
  http://127.0.0.1:10002/test/cases/pages/js/signin.js:352
TypeError: 'undefined' is not a function (evaluating 'testHelpers.testInvalidPasswordAndValidationPassword("submit with", testInvalidPasswordAndValidationPassword)')

  http://127.0.0.1:10002/test/cases/dialog/js/modules/set_password.js:94
  http://127.0.0.1:10002/test/cases/dialog/js/modules/set_password.js:104
currently unable to focus elements, focus check skipped - try focusing the unit test page
currently unable to focus elements, focus check skipped - try focusing the unit test page
currently unable to focus elements, focus check skipped - try focusing the unit test page
currently unable to focus elements, focus check skipped - try focusing the unit test page
currently unable to focus elements, focus check skipped - try focusing the unit test page
'waitFor()' finished in 45902ms.
Tests completed in 45718 milliseconds.
1371 tests of 1371 passed, 0 failed.
· · · ✓ OK » 7 honored (50.793s)
$ 
```

**Comment by jrgm** [comment]
My phantomjs version locally on osx:

```
$ phantomjs --version
1.8.0
```

This page suggests travis-ci is on 1.6.x, but I'm skeptical.
http://about.travis-ci.org/docs/user/ci-environment/#Headless-Browser-Testing-Tools

**Comment by shane-tomlinson** [comment]
This was fixed in 14535dd9b3ce506974b539a6982bc0c96886d664 in the password reset branch. Adding a separate PR for a standalone fix.
